hack/postrelease/Makefile

```diff
@@ -0,0 +1,68 @@
+include variables.mk
+
+### Binaries we need
+
+bin/yq:
+	$(info Downloading YQ from $(YQ4_URL))
+	@mkdir -p bin
+	@curl -fsSL $(YQ4_URL) -o $(YQ4)
+	@chmod +x $(YQ4)
+
+### Utility!
+
+require_var = $(if \
+                  $(filter-out undefined,$(origin $(1))),\
+                  ,\
+                  $(error The variable $(1) is required but is not defined) \
+          )
+
+require-vars-%:
+	$(foreach var,$(subst -, ,$*),$(call require_var,$(var)))
+
+### Actual rules below here!
+
+.PHONY: create_docker_container clean
+
+all: create_docker_container run_pytests_docker clean_docker_container
+
+## Setting up
+
+create_docker_container: .docker_container.stamp
+
+.docker_container.stamp: bin/yq Dockerfile config.json
+	$(info +++ Building docker container with Calico $(CALICO_VERSION), Operator $(OPERATOR_VERSION), Flannel $(FLANNEL_VERSION))
+	docker buildx build \
+		-t $(CALICO_POSTRELEASE_TEST_IMAGE)              \
+		--build-arg VERSION=$(CALICO_VERSION)            \
+		--build-arg FLANNEL_VERSION=$(FLANNEL_VERSION)   \
+		--build-arg OPERATOR_VERSION=$(OPERATOR_VERSION) \
+		.
+	@touch .docker_container.stamp
+
+## Execution
+
+run_pytests_docker: create_docker_container require-vars-QUAY_TOKEN
+	$(info +++ Running postrelease tests)
+	@docker run -it --rm                     \
+		-v .:/code                           \
+		-v /run/docker.sock:/run/docker.sock \
+		-e QUAY_TOKEN                        \
```

Comment:
Why is a quay token needed?  These images need to be publicly available, so I think we should be testing without a quay-token.